hudi-common/src/main/java/org/apache/hudi/common/table/HoodieTableConfig.java

```diff
@@ -151,6 +154,27 @@ public HoodieTableType getTableType() {
         : Option.empty();
   }
 
+  /**
+   * @return the table version from .hoodie properties file.
+   */
+  public HoodieTableVersion getHoodieTableVersionFromPropertyFile() {
+    if (props.contains(HOODIE_TABLE_VERSION_PROP_NAME)) {
+      String propValue = props.getProperty(HOODIE_TABLE_VERSION_PROP_NAME);
+      if (propValue.equals(HoodieTableVersion.ZERO_SIX_ZERO.version)) {
+        return HoodieTableVersion.ZERO_SIX_ZERO;
+      }
+    }
+    return DEFAULT_TABLE_VERSION;
+  }
+
+  /**
+   * @return the current hoodie table version.
+   */
+  public HoodieTableVersion getCurrentHoodieTableVersion() {
+    // TODO: fetch current version dynamically
```

Comment:
@vinothchandar : sorry forgot to ask this question earlier. May I know how to fetch current hoodie version in use?

Reply:
By reading `hoodie.properties` we need to treat everything below 0.6.0 as V_PRE_0.6.0 . Cannot deduce the actual jars per se. 

We can also make these version numbers 0,1,2 instead of PRE_0.6.0, 0.6.0, 0.7.0 and so on. ?

Reply:
let try to rephrase. Lets say a dataset was created in 0.6.0 and then user moves to hoodie version 0.6.1 and when he launches hoodie for first time, how in this code we will get to know that this is using version 0.6.1? hoodie.properties will be having 0.6.0 only right.

Reply:
`HoodieTableVersion` or someplace we need to ahve a `CURR_VERSION` variable that gets bumped to 0.6.1 . 

More I think about this. I think its better to name the versions 0,1,2... and so on, instead of release numbers. we may not bump this up every release . only when upgrade/downgrade is necessary.